flow360/component/volume_mesh.py

```diff
@@ -262,7 +262,7 @@ def detect(cls, file: str):
         """
         detects endianess UGRID mesh from filename
         """
-        if VolumeMeshFileFormat.detect(file) is not VolumeMeshFileFormat.UGRID:
+        if ".ugrid" not in file:
```

Comment:
why this change?

Reply:
to make the UGRIDEndianness class be independent on VolumeMeshFormat class

Reply:
why do you want to make UGRIDEndianess independent from VolumeMeshFormat? UGRIDEndianess is by nature dependent on mesh format. We we want to use it both in surface mesh and volume mesh, we can call it MeshFileFormat

Reply:
> why do you want to make UGRIDEndianess independent from VolumeMeshFormat? UGRIDEndianess is by nature dependent on mesh format. We we want to use it both in surface mesh and volume mesh, we can call it MeshFileFormat

surface mesh allows STL, but volume mesh doesn't, so we can't use a single MeshFileFormat for both surface and volume

Reply:
fixed.